src/Okta.Idx.Sdk.E2ETests/Steps/Pages/HomePageSteps.cs

```diff
@@ -0,0 +1,89 @@
+﻿using FluentAssertions;
+using Okta.Idx.Sdk.E2ETests.PageObjectModels;
+using TechTalk.SpecFlow;
+
+namespace Okta.Idx.Sdk.E2ETests.Steps.Pages
+{
+    [Binding]
+    public class HomePageSteps: BasePageSteps
+    {
+        private HomePage _homePageModel;
+        private LoginPage _loginPageModel;
+        private ResetPasswordPage _resetPasswordPageModel;
+        private RegisterPage _registerPageModel;
+
+        public HomePageSteps(ITestConfig testConfig, 
+            HomePage homePageModel, 
+            LoginPage loginPageModel,
+            ResetPasswordPage resetPasswordPageModel,
+            RegisterPage registerPageModel)
+            :base(testConfig)
+        {
+            _homePageModel = homePageModel;
+            _loginPageModel = loginPageModel;
+            _resetPasswordPageModel = resetPasswordPageModel;
+            _registerPageModel = registerPageModel;
+        }
+
+        [Given(@"Mary navigates to the Basic Login View")]
+        public void GivenMaryNavigatesToTheLoginPage()
+        {
+            _homePageModel.GoToPage();
+            _homePageModel.AssertPageOpenedAndValid();
+            _homePageModel.LoginButton.Click();
+            _loginPageModel.AssertPageOpenedAndValid();
+        }
+
+        [Then(@"she is redirected to the Root View")]
+        [Then(@"she is redirected to the Root Page is provided")]
```

Comment:
`is provided` doesn't seem to make sense in this sentence 🤔

Reply:
I agree. It's from Feature 3.1, line 19. Another place to make edits to the cucumber scenario